PowerUp/app/src/main/java/powerup/systers/com/IntroFragment6.java

```diff
@@ -0,0 +1,17 @@
+package powerup.systers.com;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+public class IntroFragment6 extends Fragment {
+    @Nullable
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View v = inflater.inflate(R.layout.intro_screen_6, container, false);
```

Comment:
Since all fragments have the exact same code except for the view that is inflated, you could instead have only one fragment and pass a parameter to its constructor with the frame you want to load, so you can load the correct one within a switch